### PR TITLE
feat(link): remove href prototype when empty

### LINKED ISSUE #2952: Can the [feature request] El link component add a property that disables the page jump function
### Existing Component
是

### Component Name
el-link

### Description
当我在el-table组件内使用el-link的时候，有些情况下并不需要它的跳转页面功能，仅作为个button按钮使用。

### PR DESCRIPTION
**Pull request by adaex**
fix #2952

* [x] Make sure you follow Element's contributing guide [English](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.en-US.md) | ([中文](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.zh-CN.md) | [Español](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.es.md) | [Français](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.fr-FR.md)).
* [x] Make sure you are merging your commits to `dev` branch.
* [x] Add some descriptions and refer to relative issues for your PR.

### CODE REVIEW

File: packages/link/src/index.vue [hunk 1]
```diff
@@ -6,7 +6,7 @@
       disabled && 'is-disabled',
       underline && !disabled && 'is-underline'
     ]"
-    :href="disabled ? null : href"
+    :href="disabled || href === '' ? null : href"
```
Comment (kooriookami) [inline packages/link/src/index.vue:9]: :href="disabled || !href ? null : href"

**Review by kooriookami — CHANGES_REQUESTED** [review]

File: packages/link/src/index.vue (hunk 1 above)
Comment (adaex) [reply packages/link/src/index.vue:9]: OK~

**Comment by adaex** [comment]
@kooriookami  Okay~

**Comment by kooriookami** [comment]
Thanks, I'm going to merge it.

**Review by kooriookami — APPROVED** [review]

**Pull request merged**